.circleci/config.yml

```diff
@@ -672,22 +672,6 @@ jobs:
             - packages/consul-ui/dist
       - run: *notify-slack-failure
 
-  # build static-assets file
-  build-static-assets:
-    docker:
-      - image: *GOLANG_IMAGE
-    steps:
-      - checkout
-      - attach_workspace:
-          at: ./pkg
-      - run: mv pkg/packages/consul-ui/dist pkg/web_ui # 'make static-assets' looks for the 'pkg/web_ui' path
-      - run: make static-assets
-      - persist_to_workspace:
-          root: .
-          paths:
-            - ./agent/uiserver/bindata_assetfs.go
-      - run: *notify-slack-failure
-
```

Comment:
This job was responsible for buiding bindata_assetfs.go and persisting it to the correct workspace path.

Reply:
Note that `pkg/packages/consul-ui/dist` is assumed to exist (it contains all the compiled ui files)